stable/sonatype-nexus/templates/_helpers.tpl

```diff
@@ -2,15 +2,33 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "name" -}}
+{{- define "nexus.name" -}}
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
 {{/*
 Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
-{{- define "fullname" -}}
+{{- define "nexus.fullname" -}}
```

Comment:
While you're at it: Please update templates to those create by `helm create` with Helm 2.8+.